src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
```

Comment:
Our main class shouldn't extend lottery, please create required object instead and use it to call required method further
```suggestion
public class Application {
    public static void main(String[] args) {
    Lottery lottery = new Lottery();
```